src/middlewares/authorization.middleware.js

```diff
@@ -0,0 +1,15 @@
+import userService from '../services/user.service';
+import ResponseService from '../services/response.service';
+
+const authorization = {
+  checkManager: async (req, res, next) => {
```

Comment:
The function name is not descriptive enough and does not precisely say what it does. Also, move it inside the auth middleware file